core/domain/classifier_domain.py

```diff
@@ -125,46 +125,123 @@ def __init__(self, job_id, algorithm_id, interaction_id, exp_id,
 
     @property
     def job_id(self):
+        """Returns the job_id of the classifier training job.
+
+        Returns:
+            str. The unique id of the classifier training job.
+        """
         return self._job_id
 
     @property
     def algorithm_id(self):
+        """Returns the algorithm_id of the algorithm used for generating
+            the classifier.
+
+        Returns:
+            str. The id of the algorithm used for generating the classifier.
+        """
         return self._algorithm_id
 
     @property
     def interaction_id(self):
+        """Returns the interaction_id to which the algorithm belongs.
+
+        Returns:
+            str. The id of the interaction to which the algorithm belongs.
+        """
         return self._interaction_id
 
     @property
     def exp_id(self):
+        """Returns the exploration id for which the classifier will be
+            generated.
+
+        Returns:
+            str. The id of the exploration that contains the state
+                for which classifier will be generated.
+        """
         return self._exp_id
 
     @property
     def exp_version(self):
+        """Returns the exploration version.
+
+        Returns:
+            str. The version of the exploration when the training job was
```

Comment:
Just to check: are you sure this is a string? How did you check? (A similar property seems to be listed as an int later on so I want to make sure.)

Reply:
Sorry, the exploration version is an integer. I verified this by `core.controllers.classfier_domain_test.py` which raises validation error when string is provided for exp_version instead of int (Line 207). Also, there were other places where it was stated as a str. I have corrected them too.  
Thanks!